spring/pim-tool-back/src/main/java/vn/elca/training/model/entity/ProjectEmployee.java

```diff
@@ -0,0 +1,53 @@
+package vn.elca.training.model.entity;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Objects;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "PROJECT_EMPLOYEE")
+public class ProjectEmployee implements Serializable {
```

Comment:
[N] If there is no other columns than Ids in the join table. Just use @ManyToMany, it makes managing the relationship much easier.